src/home.html

```diff
@@ -75,6 +75,7 @@
             <p class="error error-note"></p>
           </div>
           <div class="action-form">
+            <button class="btn-delete">delete</button>
```

Comment:
Should Delete?

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!